test/e2e/steps/sends_message.dart

```diff
@@ -48,6 +49,30 @@ final StepDefinitionGeneric sendsMessageToMe =
     ..timeout = const Duration(minutes: 5),
 );
 
+/// Sends a message from the specified [User] to the group with the provided
+/// name with the provided text.
```

Comment:
```suggestion
/// name and text.
```

Чтобы тавтологии избежать.